eclcli/sss/sssclient/common/serializer.py

```diff
@@ -3,6 +3,8 @@
 from xml.parsers import expat
 
 from oslo_serialization import jsonutils
```

Comment:
I think that it is necessary to describe like other processing.
```suggestion
try:
    from oslo_serialization import jsonutils
except ImportError:
    from oslo.serialization import jsonutils
```